app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663146331895639?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
This is resolved.